lib/internal/Magento/Framework/Reflection/Test/Unit/TypeProcessorTest.php

```diff
@@ -25,6 +27,12 @@ class TypeProcessorTest extends \PHPUnit\Framework\TestCase
     protected function setUp()
     {
         $this->_typeProcessor = new \Magento\Framework\Reflection\TypeProcessor();
+        $loader = new ClassLoader();
+        $loader->addPsr4(
```

Comment:
Why do we need to hack autoloader instead of placing fixtures accordingly?

Reply:
To do not use `_files` word in the class namespace.

Reply:
You don't need to use `_files` and do manual include. Just place fixtures in something like `lib/internal/Magento/Framework/Reflection/Test/Unit/TypeProcessorTest/Fixture/TSample.php`.

Reply:
If you open integration or unit tests you will see all fixtures located in `_files` directory. I don't want to change such "standard".

Reply:
There is no such standard. I remember such cases in integration tests just because there is no class usually.

When we can make our code `PSR-4` compliant we should do so. Please remove unnecessary `ClassLoader` complication, other `_files` occurrences containing PHP classes should be considered as technical debt.